src/components/LoansByCategory/MGDigestLightbox.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<kv-lightbox
+		:visible="isLightboxVisible"
+		title=""
+		@lightbox-closed="closeLightbox"
+	>
+		<h2 class="tw-text-h2 tw-text-center">
```

Comment:
Wrap this `h2` in a template to use it as the header `<template #header>` and gain control over how it's styled.